Update users.toml

ctanas opened the pull request.

kaushalmodi commented:
Thanks for sharing this!

May I request a bit more information on https://github.com/ctanas/parsec-source?

Prospective users checking out the Users list would want to know how ox-hugo works with Hugo.  I see that on your parsec-source repo, it does not contain the Hugo source (which at minimum needs a config.toml file and the theme, as the content is generated by ox-hugo). 

Ideally someone might try out doing a recursive clone of a repo (e.g. my site: https://gitlab.com/kaushalmodi/kaushalmodi.gitlab.io), export from the Org file and simply run Hugo). 

So if you are unable to share the source of the Hugo side of things in this repo, can you provide a brief set of instructions in the repo's README (btw GitHub renders README.org just fine; it doesn't have to be README.md :)) on how you generate your site?

But given that your Org source is very rich, I'll go ahead and submit this PR anyways. 

Thanks!

ctanas commented:
Thank you!

I have no idea why I though it's a good idea to keep the org source file and the rest of the hugo generated files in two different repos... I've moved the source file in the main repo and updated my edit to your users.toml. This way, my workflow will be better too, pushing changes from one single place. 

So now my repo contains everything: the source org file, the hugo generated files and theme files. The final version of the website is generated by Netlify, but there's a line in the README file about that. So this should probably clear all your questions :)

Best regards,
Claudiu.

On Sun, Sep 8, 2019, at 15:02, Kaushal Modi wrote:
> Thanks for sharing this!

> May I request a bit more information on https://github.com/ctanas/parsec-source?

> Prospective users checking out the Users list would want to know how ox-hugo works with Hugo. I see that on your parsec-source repo, it does not contain the Hugo source (which at minimum needs a config.toml file and the theme, as the content is generated by ox-hugo).

> Ideally someone might try out doing a recursive clone of a repo (e.g. my site: https://gitlab.com/kaushalmodi/kaushalmodi.gitlab.io), export from the Org file and simply run Hugo).

> So if you are unable to share the source of the Hugo side of things in this repo, can you provide a brief set of instructions in the repo's README (btw GitHub renders README.org just fine; it doesn't have to be README.md :)) on how you generate your site?

> But given that your Org source is very rich, I'll go ahead and submit this PR anyways.

> Thanks!

> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/kaushalmodi/ox-hugo/pull/288?email_source=notifications&email_token=AAJRTVA5GATQ7QHFOLLWENDQITSV5A5CNFSM4IUS27XKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD6FOJQQ#issuecomment-529196226>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AAJRTVA6VMSMPRTMYCYXPX3QITSV5ANCNFSM4IUS27XA>.
